src/browser.ts

```diff
@@ -117,13 +117,20 @@ class GitHubFileBrowser extends Widget {
     this._onPathChanged();
 
     this._drive.rateLimitedState.changed.connect(this._updateErrorPanel, this);
+
   }
 
   /**
    * An editable widget hosting the current user name.
    */
   readonly userName: GitHubEditableName;
 
+  /**
+   * The GitHub base URL
+   * Set by the settingsRegistry change hook
+   */
+  baseUrl: string;
```

Comment:
Good point, it needs to be settable from the outside. In that case, can we use a getter/setter as per the jupyterlab [style guide](https://github.com/jupyterlab/jupyterlab/wiki/TypeScript-Style-Guide#classes)?

Reply:
Done!